source/patterns/@aws-solutions-constructs/aws-fargate-stepfunctions/README.md

```diff
@@ -0,0 +1,142 @@
+| createCloudWatchAlarms | `boolean`|Whether to create recommended CloudWatch alarms.|
+|logGroupProps?|[`logs.LogGroupProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-logs.LogGroupProps.html)|Optional user provided props to override the default props for for the CloudWatchLogs LogGroup.|
+|stateMachineEnvironmentVariableName?|`string`|Optional name for the Step Function environment variable set for the container.|
+|stateMachineArnEnvironmentVariableName?|`string`|Optional name for the Step Function ARN environment variable set for the container.|
+
+## Pattern Properties
+
+| **Name**     | **Type**        | **Description** |
+|:-------------|:----------------|-----------------|
+| vpc | [`ec2.IVpc`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-ec2.IVpc.html) | The VPC used by the construct (whether created by the construct or provided by the client) |
+| service | [`ecs.FargateService`](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-ecs.FargateService.html) | The AWS Fargate service used by this construct (whether created by this construct or passed to this construct at initialization) |
+| container | [`ecs.ContainerDefinition`](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-ecs.ContainerDefinition.html) | The container associated with the AWS Fargate service in the service property. |
+| stateMachine| [`sfn.StateMachine`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-stepfunctions.StateMachine.html)|Returns an instance of `sfn.StateMachine` created by the construct.|
+| stateMachineLogGroup|[`logs.ILogGroup`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-logs.ILogGroup.html)|Returns an instance of the `logs.ILogGroup` created by the construct for StateMachine.|
+| cloudwatchAlarms? | [`cloudwatch.Alarm[]`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-cloudwatch.Alarm.html)|Returns a list of `cloudwatch.Alarm` created by the construct.|
+
+## Default settings
+
+Out of the box implementation of the Construct without any override will set the following defaults:
+
+### AWS Fargate Service
+* Sets up an AWS Fargate service
+  * Uses the existing service if provided
+  * Creates a new service if none provided.
+    * Service will run in isolated subnets if available, then private subnets if available and finally public subnets
+* Adds environment variables to the container with the ARN and Name of the State Machine
+* Add permissions to the container IAM role allowing it to read/write to the State Machine
+
+### AWS Step Functions
+* Sets up an AWS Step Function
+  * Uses an existing step function if one is provided, otherwise creates a new one
+* Adds an Interface Endpoint to the VPC for Step Functions (the service by default runs in Isolated or Private subnets)
+* Enables CloudWatch logging
+* Deploys best practices of CloudWatch Alarms
```

Comment:
Drop this (and we'll drop it on lambda-stepfunctions).

Actually, this section of aws-lambda-stepfunctions and aws-lambda-step-function is not very good...